spec/models/landing_page_spec.rb

```diff
@@ -42,4 +45,11 @@
       expect(described_class.new(content_item).blocks.first.type).to eq(expected_type)
     end
   end
+
+  describe "#menus" do
+    it "returns a map of menus" do
+      expect(described_class.new(content_item).navigation_groups["Top Menu"]["id"]).to eq("Top Menu")
+      expect(described_class.new(content_item).navigation_groups["Top Menu"]["links"].count).to eq(2)
```

Comment:
Shouldn't this be 4?

Reply:
This is just the direct links (of which there are two).

Reply:
Ah, perhaps the test should name should be updated to reflect that.
Is there ever a case when all 4 are returned?